program-runtime/src/compute_budget.rs

```diff
@@ -276,6 +282,15 @@ impl ComputeBudget {
             .map(|fee_type| PrioritizationFeeDetails::new(fee_type, self.compute_unit_limit))
             .unwrap_or_default())
     }
+
+    // https://github.com/solana-labs/solana/issues/31453 to round compute-unit-price, in micro-lamports,
+    // down to its nearest 1_000 micro-lamport.
+    const ROUND_DOWN_FACTOR: u64 = 1_000;
+    fn round_prioritization_fee(micro_lamports: u64) -> u64 {
+        micro_lamports
+            .saturating_div(Self::ROUND_DOWN_FACTOR)
```

Comment:
Dividing and multiplying will take many more CPU cycles. I am unsure, but maybe the CU for the compute budget should be updated.

Reply:
Good point! I'll bench the perf to see how it effects

Reply:
Meanwhile, can you check the SIMD #50 please?

Reply:
Step 1: the Average TPS, measured by `NDEBUG=1 ./multinode-demo/bench-tps.sh --use-randomized-compute-unit-price`, looks no change between master and this branch.
|master|branch|
|--------|--------|
|  39000.117 | 38451.242 |
|  38450.566 | 39375.75 |
|  38450.66 | 38179.62 |